src/main/java/roomescape/reservation/entity/Reservation.java

```diff
@@ -0,0 +1,28 @@
+package roomescape.reservation.entity;
+
+import java.time.LocalDateTime;
+
+public class Reservation {
+
+    private long id;
+    private String name;
+    private LocalDateTime dateTime;
```

Comment:
final 로 선언해서 재할당을 막아주세요!

```suggestion
    private final long id;
    private final String name;
    private final LocalDateTime dateTime;
```

Reply:
수정하였습니다!
[refactor: final을 통해 필드 재할당을 막는다.](https://github.com/woowacourse/spring-roomescape-admin/pull/209/commits/ade35232bffbf1ecebb44d4cce0dc30f1fb4d5b4)